sphinx/util/inspect.py

```diff
@@ -111,11 +111,16 @@ def getargspec(func):
                                kwonlyargs, kwdefaults, annotations)
 
 
-def unwrap(obj: Any) -> Any:
-    """Get an original object from wrapped object."""
+def unwrap_all(obj: Any) -> Any:
```

Comment:
-1 for renaming. I think python's and sphinx's are almost same.

Reply:
@tk0miya sorry for the late reply, hope you are doing well with everything that is going on in the world...

I split this out because one of the tests would break if I used this form of `unwrap` because it would take off the classmethod/staticmethod descriptors. `unwrap` from the stdlib will only take away `__wrapped__` functions which are going to only be defined if you had wrapped with `@functools.wraps` or one of the other functions that was designed to emulate a wrapped function this way. In this PR you will find both uses of `inspect.unwrap` and `inspect.unwrap_all`.

To save you some looking for the `unwrap` which is needed from the stdlib:
- https://github.com/sphinx-doc/sphinx/pull/7222/files#diff-48ca4d6544b092f35d729127500bff5fR979
- https://github.com/sphinx-doc/sphinx/pull/7222/files#diff-48ca4d6544b092f35d729127500bff5fR1341
- https://github.com/sphinx-doc/sphinx/pull/7222/files#diff-84bfd2ad46b7c8e97e5a06abb886c065R207

Reply:
Okay, I understand.